packages/scoped-elements/src/transform.js

```diff
@@ -1,8 +1,28 @@
-import { fromCache, toCache } from './cache.js';
 import { registerElement } from './scoped-elements.js';
 
-const re = /<\/?([a-zA-Z0-9-]+)/g;
+/**
+ * @description allowed tag name chars
+ * @type {string}
+ */
+const chars = `-|\\.|[0-9]|[a-z]`;
 
+/**
+ * @description Regular Expression to find a custom element tag
+ * @type {RegExp}
+ */
+const re = new RegExp(`<\\/?([a-z](${chars})*-(${chars})*)`, 'g');
```

Comment:
This regular expression avoids the processing of native tags like `button`, `p`, etc and only custom tags are going to be processed.

Reply:
It will give false positives for comments and attributes though, right?

Reply:
It will give false positives for comments and attributes though, right?

Reply:
I don't think so... in reality is searching elements like `<my-button` or `</my-button` and attributes can't start with `<` or `</`